src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -75,7 +74,7 @@ object Run  {
 
       // Update the database state:
       val newBackendInfo = JesCallBackendInfo(db.CallStatus(ExecutionStatus.Running.toString), Option(JesId(run.jesId)), Option(JesStatus(currentStatus.toString)))
-      dataAccess.updateExecutionBackendInfo(run.workflowId, run.call, newBackendInfo)
+      DataAccess.instance.updateExecutionBackendInfo(run.workflowId, run.call, newBackendInfo)
```

Comment:
Yes, I think you're right.  I'm trying to find some implicity magic to support `DataAccess.updateExecutionBackendInfo` without having to tediously make the `DataAccess` companion object a delegate by hand.